clients/admin-ui/src/features/data-discovery-and-detection/DetectionItemActions.tsx

```diff
@@ -0,0 +1,146 @@
+
+  const { diff_status: diffStatus, child_diff_statuses: childDiffStatus } =
+    resource;
+
+  // No actions for database level
+  if (resourceType === StagedResourceType.DATABASE) {
+    return null;
+  }
+
+  // We enable monitor / stop monitoring at the schema level only
+  // Tables and field levels can mute/unmute
+  const isSchemaType = resourceType === StagedResourceType.SCHEMA;
+  const isFieldType = resourceType === StagedResourceType.FIELD;
+
+  const showStartMonitoringAction =
+    (isSchemaType && diffStatus === undefined) ||
+    (!isFieldType && diffStatus === DiffStatus.ADDITION);
+  const showMuteAction = diffStatus !== DiffStatus.MUTED;
+  const showUnmuteAction = diffStatus === DiffStatus.MUTED;
+  const showConfirmAction =
+    diffStatus === DiffStatus.MONITORED &&
+    childDiffStatus &&
+    (childDiffStatus[DiffStatus.ADDITION] ||
+      childDiffStatus[DiffStatus.REMOVAL]);
+
+  const showRemoveAction = false;
+
+  return (
+    <HStack onClick={(e) => e.stopPropagation()}>
+      {showStartMonitoringAction && (
+        <ActionButton
+          title="Monitor"
+          icon={<MonitorOnIcon />}
+          onClick={async () => {
+            setIsProcessingAction(true);
+            await confirmResourceMutation({
+              staged_resource_urn: resource.urn,
+              monitor_config_id: resource.monitor_config_id!,
+            });
+            setIsProcessingAction(false);
+          }}
+          disabled={isProcessingAction}
+        />
+      )}
+      {showUnmuteAction && (
+        <ActionButton
+          title="Monitor"
+          icon={isSchemaType ? <MonitorOnIcon /> : <ViewIcon />}
+          onClick={async () => {
+            setIsProcessingAction(true);
+            await unmuteResourceMutation({
+              staged_resource_urn: resource.urn,
+            });
+            setIsProcessingAction(false);
+          }}
+          disabled={isProcessingAction}
+        />
+      )}
+      {showConfirmAction && (
+        <ActionButton
+          title="Confirm"
+          icon={<CheckIcon />}
+          onClick={async () => {
+            setIsProcessingAction(true);
+            await confirmResourceMutation({
+              staged_resource_urn: resource.urn,
+              monitor_config_id: resource.monitor_config_id!,
+            });
+            setIsProcessingAction(false);
+          }}
+          disabled={isProcessingAction}
+        />
+      )}
+      {showRemoveAction && (
+        <ActionButton
+          title="Drop"
+          icon={<TrashCanOutlineIcon />}
+          onClick={async () => {
+            setIsProcessingAction(true);
+            await confirmResourceMutation({
+              staged_resource_urn: resource.urn,
+              monitor_config_id: resource.monitor_config_id!,
+            });
+            setIsProcessingAction(false);
+          }}
+          disabled={isProcessingAction}
+        />
+      )}
+      {/* Negative actions should be last */}
```

Comment:
what's a negative action?

Reply:
I'll update the comment. "Positive Actions (Monitor, Confirm) goes first. Negative actions such as ignore should be last" does that make it more clear?

Reply:
yes that makes sense, thanks!